grid_map_core/src/GridMap.cpp

```diff
@@ -719,18 +719,18 @@ Position GridMap::getClosestPositionInMap(const Position& position) const {
   const Position3 bottomRightCorner(position_.x() - halfLengthX, position_.y() - halfLengthY, 0.0);
 
   // Find constraints.
-  const double maxX = topRightCorner.x();
+  const double maxX = topLeftCorner.x();
```

Comment:
Both should be equal. Is this change for better readability?

Reply:
Yes, indeed. I think it was clearer this way because it recalls the 2 corner points of the bounding box: _max_ values are the top-left ones while _min_ values the bottom-right's